timeboost/src/binaries/sailfish.rs

```diff
@@ -110,18 +103,11 @@ struct Cli {
     /// Backwards compatibility. This allows for a single region to run (i.e. local)
     #[clap(long, default_value_t = false)]
     multi_region: bool,
-
-    /// The number of transactions to generate per second.
-    #[clap(long, default_value_t = 100)]
-    tps: u32,
 }
 
 async fn run(
-    mut coordinator: Coordinator<SailfishBlock, Rbc<SailfishBlock>>,
-    producer: TransactionQueue,
-    mut interval: Interval,
-    handles: Vec<JoinHandle<()>>,
-    #[cfg(feature = "until")] mut until_handle: JoinHandle<()>,
+    mut coordinator: Coordinator<Unit, Rbc<Unit>>,
```

Comment:
@jparr721: Since `SailfishBlock` does not exist anymore I use `Unit` here which is quite empty, so benchmarking sailfish tests with empty blocks. If this is not acceptable, I could devise a new `SailfishBlock`-like type here with some length.

Reply:
We might still want to have some sort of block so I can benchmark with data. Could we perhaps just define it in this file and use it?

Reply:
3bb661965eced896c83ff50f62539565f61f8b85 defines a simple 512-bytes block.